src/components/Popover/__stories__/Popover.stories.tsx

```diff
@@ -93,9 +96,14 @@ export const FocusManagement: Story = {
     args: {
         ...Default.args,
         content: (
-            <div style={{padding: 10}}>
-                Content with <Link href="#">Link</Link> and <Button>Button</Button>
-            </div>
+            <Alert
```

Comment:
It looks broken with Alert styling:
<img width="181" alt="" src="https://github.com/user-attachments/assets/2715e23f-5e30-4638-90d8-769e9e9569df" />